locale/en/docs/guides/index.md

```diff
@@ -29,3 +29,4 @@ layout: docs.hbs
 - [Backpressuring in Streams](backpressuring-in-streams/)
 - [Domain Module Postmortem](domain-postmortem/)
 - [How to publish N-API package](publishing-napi-modules/)
+- [ABI Stability](abi-stability/)
```

Comment:
Is this really a module-related guide? If not, does it maybe belong in the General section instead of here?

Reply:
Well, although it starts with generic ABI-related information, it mostly pertains to N-API. That's why I put it there.

Reply:
Works for me.